pkg/resource/deploy/source_eval_test.go

```diff
@@ -2042,47 +2042,46 @@ func (c *configSourceMock) GetPackageConfig(pkg tokens.Package) (resource.Proper
 
 func TestDefaultProviders(t *testing.T) {
 	t.Parallel()
-	t.Run("newRegisterDefaultProviderEvent", func(t *testing.T) {
+	t.Run("normalizeProviderRequest", func(t *testing.T) {
 		t.Parallel()
-		t.Run("error in GetPackageConfig()", func(t *testing.T) {
+		t.Run("use defaultProvider", func(t *testing.T) {
 			t.Parallel()
-			expectedErr := errors.New("expected error")
+			v1 := semver.MustParse("0.1.0")
 			d := &defaultProviders{
+				defaultProviderInfo: map[tokens.Package]workspace.PluginSpec{
+					tokens.Package(""): {
```

Comment:
bit confusing this being an empty string, can we change this to "pkg" and fill in the `providers.ProviderRequest` below to say "pkg" as well.

Reply:
That's what the existing test code did but I'll change it.

Reply:
Thanks